src/libsync/propagatedownload.cpp

```diff
@@ -631,6 +673,108 @@ void PropagateDownloadFile::slotGetFinished()
     validator->start(_tmpFile.fileName(), checksumHeader);
 }
 
+void PropagateDownloadFile::slotZsyncGetFinished()
+{
+    propagator()->_activeJobList.removeOne(this);
+
+    GETFileZsyncJob *job = qobject_cast<GETFileZsyncJob *>(sender());
+    ASSERT(job);
+
+    SyncFileItem::Status status = job->errorStatus();
+    if (status != SyncFileItem::NoStatus) {
+        done(status, job->errorString());
+        return;
+    }
+
+    if (!job->reply()) {
```

Comment:
unnecessary check unless something weird is going on in GetFileZsyncJob

Reply:
I think this was left-over from old debug. Removed.